plugins/module_utils/vmware.py

```diff
@@ -50,8 +50,63 @@ class TaskError(Exception):
     def __init__(self, *args, **kwargs):
         super(TaskError, self).__init__(*args, **kwargs)
 
+def check_answer_question_status(vm):
```

Comment:
```suggestion

def check_answer_question_status(vm):
```

linters: E302 expected 2 blank lines, found 1